package.json

```diff
@@ -36,7 +36,8 @@
     "typedoc": "typedoc",
     "typedoc:packages": "lerna run --parallel typedoc",
     "audit:public": "npm audit --registry https://registry.npmjs.org/",
-    "prepare": "husky install"
+    "prepare": "husky install",
+    "prepack": "node scripts/prepareLicenses.js"
```

Comment:
Should the `prepack` script be removed from here, to remind us not to run `npm pack` at the top level?

Reply:
The `npm publish` command in the Jenkins pipeline only runs in the lower-level package directories, not at the top level, but it shouldn't hurt anything to leave the prepack script here.